File: Java/CodingBat/countCode.java
// http://codingbat.com/prob/p123614

public int countCode(String str) {
  int numberOfValidCodeInstances = 0;
  for (int i = 0; i < str.length()-3; ++i) {
    if (str.substring(i, i+2).equals("co") && str.charAt(i+3) == 'e') {
      numberOfValidCodeInstances++;
    }
  }
  return numberOfValidCodeInstances;
}

File: Python/CodingBat/count_code.py
# http://codingbat.com/prob/p186048

def count_code(str):
  code_count = 0
  
  for i in range(len(str)-3):
    if str[i:i+2] == "co" and str[i+3:i+4] == 'e':
      code_count += 1
      
  return code_count
